ska_dbi/sqsh.py

```diff
@@ -0,0 +1,93 @@
+import subprocess
+from pathlib import Path
+from astropy.table import Table
+
+SYBASE = "/soft/SYBASE16.0"
+ENV_VARS = [
+    f"SYBASE={SYBASE}",
+    "LD_LIBRARY_PATH=/soft/SYBASE16.0/OCS-16_0/lib:/soft/SYBASE16.0/OCS-16_0/lib3p64:/soft/SYBASE16.0/OCS-16_0/lib3p",
+]
+SQSH_BIN = "/usr/local/bin/sqsh.bin"
+
+
+class Sqsh(object):
+    def __init__(
+        self,
+        server=None,
+        user=None,
+        passwd=None,
+        database=None,
+        sqshrc=None,
+        authdir="/proj/sot/ska/data/aspect_authorization",
+        **kwargs,
+    ):
+        DEFAULTS = {"server": "sqlsao", "user": "aca_ops", "database": "axafapstat"}
+
+        self.server = server or DEFAULTS.get("server")
+        self.user = user or DEFAULTS.get("user")
+        self.database = database or DEFAULTS.get("database")
+        self.passwd = passwd
+        self.sqshrc = sqshrc
+
+        if not Path(SYBASE).exists():
+            raise RuntimeError("SYBASE does not exist: %s" % SYBASE)
+        if not Path(SQSH_BIN).exists():
+            raise RuntimeError("sqsh.bin does not exist: %s" % SQSH_BIN)
+
+        if self.passwd is None and self.sqshrc is None:
+            sqshrc_file = (
+                Path(authdir) / f"sqsh-{self.server}-{self.database}-{self.user}"
```

Comment:
The idea here is that this access would still use files in aspect_authorization, but they are different files (start with sqsh) with a two-line format that sets username and password in sqshrc style.

Reply:
Looks good. I think for this purpose that having the `authdir` `sqsh-*` files as the only option seems sufficient, but maybe you had other use cases in mind.